avalon/houdini/pipeline.py

```diff
@@ -313,16 +311,15 @@ def process(self):
         return instance
 
 
-def _on_scene_open(*args):
-    api.emit("open", *[])
-
-
-def _on_scene_new(*args):
-    api.emit("new", *[])
-
-
-def _on_scene_save(*args):
-    api.emit("save", *[])
+def on_file_event_callback(event):
```

Comment:
Actually made Houdini callbacks work - wait... aren't these already in core master?